packages/koa-shopify-auth/src/auth/client/request-storage-access.ts

```diff
@@ -4,10 +4,10 @@ const requestStorageAccess = (shop: string, prefix = '/') => {
       function redirect() {
         var targetInfo = {
           myshopifyUrl: "https://${encodeURIComponent(shop)}",
-          hasStorageAccessUrl: "${prefix}auth/inline?shop=${encodeURIComponent(
+          hasStorageAccessUrl: "${prefix}/auth/inline?shop=${encodeURIComponent(
```

Comment:
`prefix` defaults to `/` so in the case where a prefix isn't passed in, this would now become `//auth/incline...`
For this fix, we'll want to set the default `prefix = ''` in the function args. 

Also, we'll need to apply this change to `appTargetUrl` below. 

@dooman87, does this make sense given your previous PR?

Reply:
Sorry, missed this one @ismail-syed.

Looks like I messed it up in my previous one. And it works in my app cause I used my forked version before the latest change to default value of `prefix` :( 

I think you are right and we need to remove default value from prefix. `appTargetUrl` looks alright to me cause it's the last item in the path. 

I'm happy to add a unit test for this bit. What would I use to write it? I saw in the other parts jest was used. Is it the way to go?

Reply:
> What would I use to write it? I saw in the other parts jest was used. Is it the way to go?

Yes, jest is the way to go. Thanks!

Reply:
@dooman87 @ismail-syed can we fix this? it makes using a prefix unusable, which is even worse as the example usage of using this package is with a prefix.

Reply:
@ismail-syed apologies, here in Melbourne, we went into stage 4 lockdown which means no childcare. It's going to finish in 2 weeks in best case scenario. Could we merge this one please and cut the release. I think functionality wise it's good to go. I have unit tests on my list and will add them.

Reply:
Follow-up: https://github.com/Shopify/quilt/issues/1550#issuecomment-683845036